src/fides/api/ops/schemas/saas/saas_config.py

```diff
@@ -150,8 +151,13 @@ def validate_grouped_inputs(cls, values: Dict[str, Any]) -> Dict[str, Any]:
                             "Grouped input fields must either be reference fields or identity fields."
                         )
                     if param.references:
-                        collect = param.references[0].field.split(".")[0]
-                        referenced_collections.append(collect)
+                        # reference may be a str, in which case it's an external reference.
+                        # since external references are parameterized via secrets,
+                        # they cannot be resolved and checked at this point in the validation.
+                        # so here we only perform the check if the reference is a FidesopsDatasetReference
+                        if isinstance(param.references[0], FidesopsDatasetReference):
+                            collect = param.references[0].field.split(".")[0]
+                            referenced_collections.append(collect)
```

Comment:
we're not - i _think_ adding one of these external references as a `grouped_input` is considered out of scope for the initial version of this feature, especially since `grouped_input`s need to be from the same collection? @galvana curious on your thoughts.

if indeed it's out of scope currently, then perhaps we should explicitly fail validation here if we find an external reference as a `grouped_input`, for clarity.

Reply:
added logic to throw a validation error if we encounter an external reference as a `grouped_input`: https://github.com/ethyca/fides/pull/1269/files#diff-fb8ec5e3fd765b301329795f257fabe2d8763f22822f7876480519fe8598b4b9R162-R165